### PR TITLE
Benchmark scores update

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- update scores after re-markup

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -331,3 +334,4 @@ jobs:
         exit ${exit_code}
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
```
Comment (babenek) [inline .github/workflows/benchmark.yml:337]:
```suggestion
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -166,7 +168,8 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v3
       with:
-        repository: Samsung/CredData
+        repository: babenek/CredData
+        ref: auxiliary
```
Comment (babenek) [inline .github/workflows/benchmark.yml:172]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -62,7 +63,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:67]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 4]
```diff
@@ -22,7 +22,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:26]:
```suggestion
        repository: Samsung/CredData
```

**Review by babenek — COMMENTED** [review]
TODO: rollback after https://github.com/Samsung/CredData/pull/70

**Review by Yullia — DISMISSED** [review]
Approved

**Review by xDizzix — APPROVED** [review]

**Review by csh519 — APPROVED** [review]
Thank you for the re-mark and benchmark testing.

LGTM :+1:

**Pull request merged**